PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,6 @@
 <!--
   If you have a relevant JIRA issue number, please put it in the issue title.
   Example: MDEXP-6 Create Inventory Instance to MARC Bib record mapping rules
-
```

Comment:
Did you mean to change the PULL_REQUEST_TEMPLATE in this PR?  I don't see it mentioned anywhere in the PR description.  These changes, while simple seem out of place here, especially given the lack of rational behind them.

Reply:
When I did the PR the template didn't exist, but at this moment another person committed it in the master. Have I to remove it?

Reply:
no, it's fine - was just curious what the story was